src/app/post/[postId]/desktop/error.tsx

```
@@ -0,0 +1,15 @@
+'use client';
+
+import { commonStatusLayout, commonStatusText } from '@/app/common-status.css';
+
+export default function Error() {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)